src/librand/distributions/ziggurat_tables.rs

```diff
@@ -12,8 +12,8 @@
 // algorithm. Autogenerated by `ziggurat_tables.py`.
 
 pub type ZigTable = &'static [f64; 257];
-pub static ZIG_NORM_R: f64 = 3.654152885361008796;
-pub static ZIG_NORM_X: [f64; 257] =
```

Comment:
Right, but it can still go into the metadata and is also arguably more roundabout (e.g. it's not clear that this is better than a `static`)